deploy/lib/server_config.rb

```diff
@@ -112,7 +112,7 @@ def info
       logger.info "<info>"
       logger.info "\s\s<isJar>#{@@is_jar}</isJar>"
       logger.info "\s\s<properties>"
-      @properties.sort {|x,y| y <=> x}.each do |k, v|
+      @properties.sort {|x,y| x <=> y}.each do |k, v|
```

Comment:
You are correct. Descending order is necessary for replacement, but these lines concern display/output only. Ascending order reads much easier..

I'll rebase and update the comments..